src/schemas/json/jsconfig.json

```
@@ -207,6 +207,16 @@
           "type": "object",
           "description": "Instructs the TypeScript compiler how to compile .ts files.",
           "properties": {
+            "allowArbitraryExtensions": {
+              "description": "Enable importing files with any extension, provided a declaration file is present.",
```

```suggestion
              "description": "Whether to enable importing files with any extension, provided a declaration file is present.",
```

Optional fix: descriptions for keys representing boolean options can begin with `Whether to`.

---

That is a good standard for booleans, but I think it is more important for the descriptions to match the command-line help text produced by `tsc`, even if it's non-standard.